### PR TITLE
Code Changes By Pallavi

### PR DESCRIPTION
**Pull request by PM-PallaviD**
Changes in Header And Footer in Typo Content

### CODE REVIEW

File: root/index.html [hunk 1]
```diff
@@ -37,13 +37,13 @@
             <a class="nav-link active" aria-current="page" href="#">Home</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#">Why Choose us</a>
+            <a class="nav-link" href="#">Why Choose us ?</a>
```
Comment (Akshay-Warule) [inline root/index.html]: no need of question mark

File: root/index.html [hunk 2]
```diff
@@ -37,13 +37,13 @@
...
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#">Blog </a>
+            <a class="nav-link" href="#">  New Blog </a>
```
Comment (SujataGaikwad100) [inline root/index.html]: Add s at the end of blog

File: root/index.html (hunk 1 above)
Comment (PM-PallaviD) [reply root/index.html]: resolve issue as per your suggestions

File: root/index.html (hunk 2 above)
Comment (PM-PallaviD) [reply root/index.html]: resolve issue as per your suggestions

**Review by SujataGaikwad100 — APPROVED** [review]

**Review by Akshay-Warule — APPROVED** [review]

**Pull request merged**